prog/dagorInclude/math/dag_color.h

```diff
@@ -115,44 +95,24 @@ struct Color4
 
   colorinline void clamp0()
   {
-    if (r < 0)
-      r = 0;
-    if (g < 0)
-      g = 0;
-    if (b < 0)
-      b = 0;
-    if (a < 0)
-      a = 0;
+    r = min(0.0f, r);
+    g = min(0.0f, g);
+    b = min(0.0f, b);
+    a = min(0.0f, a);
   }
   colorinline void clamp1()
   {
-    if (r > 1)
-      r = 1;
-    if (g > 1)
-      g = 1;
-    if (b > 1)
-      b = 1;
-    if (a > 1)
-      a = 1;
+    r = max(1.0f, r);
+    g = max(1.0f, g);
+    b = max(1.0f, b);
+    a = max(1.0f, a);
```

Comment:
Please make sure that all compilers are smart enough to optimize the identity write out (by looking at asm), otherwise this change can make perf worse.